management-core/src/main/java/com/ns/task/repositories/ProductRepository.java

```diff
@@ -1,9 +1,15 @@
 package com.ns.task.repositories;
```

Comment:
Since this is used for testing scope, can you try to move it to the test folder?

Reply:
I am not sure if I understand, do you refer to move to common folder? because this interface is used for service layer

Reply:
I was talking about this class only.
You're using this interface for testing purpose, but I don't know if Spring can load it fine

Reply:
This one is closed because not apply